src/accordionComponents/FourUp.js

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
-import { Box, Center, Image, VStack } from "@chakra-ui/react";
+import { Box, Center, Image, VStack } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
 
 class FourUp extends React.Component {
     render() {
-        const { fourUpName, caption } = this.props
+        const { fourUpName, caption } = this.props;
         return (
             <Center p="5">
-                <VStack> 
+                <VStack>
                     <Box width="100%">
-                        <Image height="400" src={process.env.PUBLIC_URL + `/images/${fourUpName}.png`} alt={fourUpName} />
-                    </Box>   
-                    <Box p="5"> 
+                        <Image
+                            height="400"
+                            src={`${process.env.PUBLIC_URL}/images/${fourUpName}.png`}
+                            alt={fourUpName}
+                        />
+                    </Box>
+                    <Box p="5">
                         <span style={{ textAlign: 'center' }}>{caption}</span>
-                    </Box>  
-                </VStack> 
+                    </Box>
+                </VStack>
             </Center>
         );
     }
 }
 
-export default FourUp;
\ No newline at end of file
+FourUp.propTypes = {
+    fourUpName: PropTypes.string.isRequired,
+    caption: PropTypes.string.isRequired,
+};
```

Comment:
Just so I understand what's going on here: this is specifying that these props are required when instantiating a `FourUp` component?

Reply:
Yes, there's also another class level field (in this case it would be `FourUp.defaultProps = {...}` that allows specifying default values for unrequired props. This is required in the current linting rules for all nonrequired props in order to prevent issues with `undefined` showing up as a prop where you're not expecting it.